tests/forcefields/test_jobs.py

```diff
@@ -236,7 +242,6 @@ def test_nequip_relax_maker(
             "model_path": test_dir / "forcefields" / "nequip" / "nequip_ff_sr_ti_o3.pth"
         },
     ).make(sr_ti_o3_structure)
-
     # run the flow or job and ensure that it finished running successfully
```

Comment:
i think we're missing a new assert here for the spacegroup of the relaxed structure?

Reply:
I added the fix symmetry options to the test_nequip_relaxmaker to confirm the functionality of providing the option to the RelaxMaker. We can add the assert symmetry but the problem is this STO force field just goes to cubic anyway so the symmetry was staying the same (or higher symmetry) with and without the fix symmetry option.
So I used this test case instead in the test_utils that I think you removed when merging 
```
@pytest.mark.parametrize(("fix_symmetry"), [(True), (False)])
def test_fix_symmetry(fix_symmetry):
    # adapted from the example at https://wiki.fysik.dtu.dk/ase/ase/constraints.html#the-fixsymmetry-class
    relaxer = Relaxer(
        calculator=LennardJones(), relax_cell=True, fix_symmetry=fix_symmetry
    )
    atoms_al = bulk("Al", "bcc", a=2 / 3**0.5, cubic=True)
    atoms_al = atoms_al * [2, 2, 2]
    atoms_al.positions[0, 0] += 1.0e-7
    symmetry_init = check_symmetry(atoms_al, 1.0e-6, verbose=True)
    final_struct = relaxer.relax(atoms=atoms_al)["final_structure"]
    symmetry_final = check_symmetry(
        AseAtomsAdaptor.get_atoms(final_struct), 1.0e-6, verbose=True
    )
    if fix_symmetry:
        assert symmetry_init["number"] == symmetry_final["number"]
    else:
        assert symmetry_init["number"] != symmetry_final["number"]
```
We can add the assert statement concerning the space group to the nequip test but we also need  a test case like the one above where the symmetry actually changes.

Reply:
oops, was a big hasty here and just ran `git merge -X theirs main`. didn't notice this swallowed the test 😅. now restored

Reply:
Right now we have a test for chgnet, mace and the relaxer itself so I consider this is resolved as well.